src/Type/InterfaceType/ContentNode.php

```diff
@@ -60,6 +132,20 @@ public static function register_type( TypeRegistry $type_registry ) {
 					'enqueuedStylesheets' => [
 						'toType'  => 'EnqueuedStylesheet',
 						'resolve' => static function ( $source, $args, $context, $info ) {
+							// Simulate WP template rendering
+							ob_start();
+							wp_head();
```

Comment:
This will be incompatible with eventual FSE support, and break several existing WPGraphQL extensions (e.g. seo). I haven't reviewed the PR for real, so without knowing the full goal I gotta hope there's a different way to accomplish this.

Reply:
No, it shouldn't. I've tested quite extensively at this point.

Reply:
Also it runs on the resolvers for both `enqueuedStyles` and `enqueuedScripts`, which means these `do_action()` and `apply_filters()` are getting triggered multiple times too... What's the performance hit with this change?

Reply:
Nothing that I've noticed, but I'm just using WC and some WC premium plugins, and while WC emits a lot of localize scripts for their pages I've haven't seen any slow downs in the query speed due to this change.